scripts/esbuild.mjs

```diff
@@ -0,0 +1,18 @@
+import { posix } from 'path';
+import { globSync } from 'glob';
+import { build } from 'esbuild';
+
+const cwd = process.cwd();
+const dist = posix.join(cwd, 'dist');
+const pattern = posix.join(cwd, 'src/**/*.{js,ts}');
```

Comment:
Should this instead read the include/excludes from `tsconfig.json`?

Reply:
3f5f368b197226466f38ba3a237ad6745edf0156